package.json

```diff
@@ -30,12 +30,13 @@
     "helmet": "^7.1.0",
     "module-alias": "^2.2.3",
     "mongoose": "^8.0.3",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "nodemon": "^3.0.2"
```

Comment:
nodemon isn't required, the use `npm run dev` to start the dev server and make changes while it's running to get a live preview.